pkg/util/table.go

```diff
@@ -85,6 +111,13 @@ func (t *Table) GetPointRange(key string) kv.KeyRange {
 	return kv.KeyRange{StartKey: startKey, EndKey: startKey.PrefixNext()}
 }
 
+func (t *Table) GetScanRange(key string) kv.KeyRange {
+	startKey := tablecodec.EncodeRowKey(t.tableId, []byte(key))
+	nextTableId := t.tableId + 1
+	endKey := tablecodec.EncodeRowKeyWithHandle(nextTableId, math.MinInt64)
```

Comment:
@zz-jason PTAL, I'm not sure about the scan range.

Reply:
how about:
```go
endKey := tablecodec.EncodeRowKeyWithHandle(t.tableId, math.MaxInt64)
endKey = endKey.PrefixNext()
```

Reply:
The bytes order of the `endkey` is greater than all key in the table which we want to scan, so we can scan all data in the table from startKey.
that is make sure by the TiDB encoding function:
```
// EncodeInt appends the encoded value to slice b and returns the appended slice.
// EncodeInt guarantees that the encoded value is in ascending order for comparison.
func EncodeInt(b []byte, v int64) []byte {
	var data [8]byte
	u := EncodeIntToCmpUint(v)
	binary.BigEndian.PutUint64(data[:], u)
	return append(b, data[:]...)
}
```